unity-renderer/Assets/UIComponents/Scripts/Components/Carousel/CarouselComponentView.cs

```diff
@@ -265,13 +268,16 @@ public void SetItems(List<BaseComponentView> items)
         GenerateDotsSelector();
     }
 
-    public void AddItem(BaseComponentView item)
+    public void AddItemWithDotsSelector(BaseComponentView item)
     {
         CreateItem(item, $"Item{instantiatedItems.Count}");
         SetManualControlsActive(model.showManualControls);
         GenerateDotsSelector();
     }
-
+    
+    public void AddItem(BaseComponentView item) => 
```

Comment:
We should add the `AddItem` method into the `ICarouselComponentView` interface, right?